source/MaterialXRuntime/Private/PvtApi.h

```diff
@@ -236,56 +292,68 @@ class PvtApi
         return _implementationSearchPaths;
     }
 
-    void createLibrary(const RtToken& name);
-
-    void loadLibrary(const RtToken& name, const RtReadOptions& options);
-
+    RtStagePtr loadLibrary(const RtToken& name, const FilePath& path, const RtReadOptions* options = nullptr, bool forceReload = false);
     void unloadLibrary(const RtToken& name);
+    void unloadLibraries();
 
-    RtStagePtr getLibrary(const RtToken& name)
+    size_t numLibraries() const
```

Comment:
Think I prefer libraryCount() as a name.

Reply:
This naming convention (numLibraries, numNodeDefs, numNodeGraphs, etc.) is the convention used through out the whole runtime library. If we change it I suppose we should change it everywhere to be consistent.